docs/version0.9.md

```diff
@@ -22,12 +22,108 @@
+`gencon` garbage collection policy. From v0.9.0, this feature is now available on Linux for IBM POWER<sup>&reg;</sup> and IBM Z<sup>&reg;</sup> architectures.
+For more information about this feature, see the following command line options, which control this
+behavior:
+
+- [-XX:\[+|-\]IdleTuningCompactOnIdle](xxidletuningcompactonidle.md)
+- [-XX:\[+|-\]IdleTuningGcOnIdle](xxidletuninggconidle.md)
+- [-XX:IdleTuningMinIdleWaitTime](xxidletuningminidlewaittime.md)
+- [-XX:IdleTuningMinFreeHeapOnIdle](xxidletuningminfreeheaponidle.md)
+
+The following blog post describes the benefits of using this feature: [Are you still paying for unused memory when your Java app is idle?](https://developer.ibm.com/javasdk/2017/09/25/still-paying-unused-memory-java-app-idle/)
+
+
+## New GC policy
+
+A new GC policy is introduced for [JEP 318: Epsilon: A No-Op Garbage Collector](http://openjdk.java.net/jeps/318).
+
+When this policy is enabled, the Java object heap is expanded in the normal way until the limit is
+reached, but memory is not reclaimed through garbage collection. When the limit is reached the VM shuts down.
+
+This JEP has a number of use cases including short-lived applications and certain test scenarios.
+
+To enable this policy you can use one of the following options:
+
+- [-Xgcpolicy:nogc](#xgcpolicy.md)
+- [-XX:+UseNoGC]()
+
+## Modifying the default Java heap size for applications that run in containers
+
+When using container technology, applications are typically run on their own and do not need to compete for memory. In this release, changes
+are introduced to detect when OpenJ9 is running inside a container. If your application is running in a container and
+you want the VM to allocate a larger fraction of memory to the Java heap, set the `-XX:+UseContainerSupport` option on the command line.
+
+The following table shows the maximum Java heap size that gets set, depending on the memory available:
+
+
+| Physical memory `<size>`   | Maximum Java heap size (v0.9.0) |
+|----------------------------|---------------------------------|
+| Less than 1 GB             | 50% `<size>`                    |
+| 1 GB - 2 GB                | `<size>` - 512M                 |
+| Greater than 2 GB          | 75% `<size>`                    |
+
+
+## Specifying the maximum Java heap size as a percentage value
+
+OpenJ9 now supports setting the heap size as a percentage of the physical memory. The following OpenJDK options are recognized and can be
+set for the VM:
+
+- ![Start of content that applies to Java 8 and later](cr/java8plus.png) [-XX:MaxRAMPercentage](add link when topic available) ![End of content that applies to Java 8 and later](cr/java_close_lts.png)
+- ![Start of content that applies only to Java 9 and later](cr/java9plus.png) [-XX:InitialRAMPercentage](add link when topic available) ![End of content that applies only to Java 9 or later](cr/java_close.png)
```

Comment:
-XX:InitialRAMPercentage affects Java 8 as well

Reply:
The options are `-XX:MaxRAMPercentage=<percent>` and `-XX:InitialRAMPercentage=<percent>`. Where `<percent>` is a number between 1 and 100, although not all values are feasible since they would make the heap size too small or too large for the JVM to start.

Reply:
I misunderstood a comment from Dmitri - am removing the icons now that you've explained that they are both applicable to Java 8 and later.  I've yet to add the 2 new -XX options into the user docs. I'll put the syntax information into those topics and link to them from the what's new. I'll make sure I re-use your explanations. Thanks.

Your comment below also incorporated re. behaviour when running in containers.